### PR TITLE
path

### PR DESCRIPTION
**Pull request by izhuxiaoqing**

### CODE REVIEW

File: docs-2.0/1.introduction/2.1.path.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Path types
+
+In graph theory, a path in a graph is a finite or infinite sequence of edges which joins a sequence of vertices. Paths are fundamental concepts of graph theory.
+
+Paths can be categorized into 3 types: `walk`、`trail`、`path`. For more information, see [wikipedia](https://en.wikipedia.org/wiki/Path_(graph_theory)#Walk,_trail,_path).
+
+The following picture is an example for a brief introduction.
+
+![path graph](../20.appendix/path.png)
```
Comment (whitewum) [inline docs-2.0/1.introduction/2.1.path.md:9]: you might forget copy this file.

File: docs-2.0/1.introduction/2.1.path.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+In the above path graph, edges cannot be repeatedly visited. So, this path graph contains finite paths. The longest path in this graph consists of 5 edges: `A->B->C->D->E->C`.
+
+!!! note
+
+    `MATCH`, `FIND PATH`, and `GET SUBGRAPH` statements use `trail`.
+
+There are two special cases of trail, `cycle` and `circuit`. The following picture is an example for a brief introduction.
+
+![trail graph](../20.appendix/Circuits.png)
```
Comment (whitewum) [inline docs-2.0/1.introduction/2.1.path.md:33]: ditto

**Review by whitewum — CHANGES_REQUESTED** [review]
2 figs ...

File: docs-2.0/1.introduction/2.1.path.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+!!! note
+
+    `GO` statements use `walk`.
+
+## trail
+
+A `trail` is a finite sequence of edges. Only vertices can be repeatedly visited in graph traversal. The Seven Bridges of Königsberg is a typical `trail`.
+
+In the above path graph, edges cannot be repeatedly visited. So, this path graph contains finite paths. The longest path in this graph consists of 5 edges: `A->B->C->D->E->C`.
```
Comment (randomJoe211) [inline docs-2.0/1.introduction/2.1.path.md:25]: There is no such thing as a path graph. Let's make it clear:

* A graph is a graph in graph theory. Graphs have vertices and edges.
* A picture or figure is, well, something like an image or a drawing.

So, maybe we should use "In the above picture/figure"? WDYT?

File: docs-2.0/1.introduction/2.1.path.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+   A `cycle` refers to a closed `trail`. Only the terminal vertices can be repeatedly visited. The longest path in this graph consists of 3 edges: `A->B->C->A` or `C->D->E->C`.
+
+- circuit
+
+   A `circuit` refers to a closed `trail`. Edges cannot be repeatedly visited in graph traversal. Apart from the terminal vertices, other vertices can also be repeatedly visited. The longest path in this graph: `A->B->C->D->E->C->A`.
+
+## path
+
+A `path` is a finite sequence of edges. Neither vertices or edges can be repeatedly visited in graph traversal.
```
Comment (randomJoe211) [inline docs-2.0/1.introduction/2.1.path.md:45]: Neither vertices nor edges

File: docs-2.0/1.introduction/2.1.path.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+So, the above path graph contains finite paths. The longest path in this graph consists of 4 edges: `A->B->C->D->E`.
```
Comment (randomJoe211) [inline docs-2.0/1.introduction/2.1.path.md:47]: Not path graph.

Check through the topic plz.

File: docs-2.0/1.introduction/2.1.path.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+A `trail` is a finite sequence of edges. Only vertices can be repeatedly visited in graph traversal. The Seven Bridges of Königsberg is a typical `trail`.
+
+In the above path graph, edges cannot be repeatedly visited. So, this path graph contains finite paths. The longest path in this graph consists of 5 edges: `A->B->C->D->E->C`.
+
+!!! note
+
+    `MATCH`, `FIND PATH`, and `GET SUBGRAPH` statements use `trail`.
+
+There are two special cases of trail, `cycle` and `circuit`. The following picture is an example for a brief introduction.
```
Comment (randomJoe211) [inline docs-2.0/1.introduction/2.1.path.md:31]: , and `circuit`

**Review by randomJoe211 — DISMISSED** [review]

**Review by randomJoe211 — DISMISSED** [review]

**Review by FengUyu — DISMISSED** [review]

**Comment by whitewum** [comment]
waiting for the png files being moved

**Review by cooper-lzy — APPROVED** [review]

**Review by whitewum — APPROVED** [review]

**Pull request merged**